src/App.js

```diff
@@ -1,37 +1,17 @@
 import React from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
```

Comment:
When I pulled your project I am unable to get it to start without making a quick edit here in the capitalization

`import TodoList from './todoList';`

Convention for component files is to be capitalized and the import has to match the filename or it won't work. Changing it now is sort of weird though because it must be done on GitHub as just changing it in VSCode and pushing it up doesn't change the case of file names. Let me know if you want help doing that.

Reply:
Hi Geo, thank you so much for your feedback! So in this one I have to write code exactly how you wrote it right? With the last todoListItem with the first "t" lower case?

Reply:
Ok so to break down imports a bit, here's how they work:

`import` (The exact name of the exported function) `from` (The exact name of the file)

So it's convention to have the function name be the exact same as the file name. GitHub is weird and has a weird bug(?) where if you name something `todoList.js` and commit it and push it, then later edit it to `TodoList.js` and push it up, GitHub doesn't recognize the change in case and it must be fixed manually on GitHub.

And to circle back, to be within proper convention you'll want to make the final name of that file be `TodoList.js`, update it in your code and then edit it on GitHub.